[#800] Added a reference to bats-helpers library to docs.

AlexSkrypnyk opened the pull request:
A list of provided assertions: https://bats-helpers.drevops.com/assertions

Data provider: https://bats-helpers.drevops.com/data-provider

Helpers: https://bats-helpers.drevops.com/helpers

- [x] I have reviewed the [Contributor Guidelines][contributor].
- [x] I have reviewed the [Code of Conduct][coc] and agree to abide by it

[contributor]: https://github.com/bats-core/bats-core/blob/master/docs/CONTRIBUTING.md
[coc]:         https://github.com/bats-core/bats-core/blob/master/docs/CODE_OF_CONDUCT.md

martin-schulze-vireso commented:
As discussed in #1025, I would prefer if we change this section to link to https://github.com/bats-core/bats-core/wiki/Libraries-For-Bats

AlexSkrypnyk commented:
@martin-schulze-vireso
With all due respect, I’m finding it a bit disheartening that my library appears to be overlooked—despite following the contribution process thoroughly, including opening discussions, submitting a PR, and addressing feedback. Meanwhile, I’ve noticed that some libraries listed have not been maintained for years.

Is there a specific reason my library isn’t being considered? If there’s something I’ve missed or could improve, I’d really appreciate your guidance. I believe it brings genuinely helpful functionality to the community and would love to understand how we can move forward.

Thanks again for your time and consideration.

martin-schulze-vireso commented:
I am sorry that you feel overlooked that was not my intention.

As I already said and tried to make clear by the edits to this PR and Theo linked Wiki page, we don't have enough resources to curate a vetted list of third party libraries and I want to avoid users getting this impression by including them in the official documentation. Our official libraries are maintained by us alongside current core and reviewed to prevent malicious actors from injecting code. This clearly distinguishes them from the third party libraries.

I thought offering the Wiki page and link to it from the documentation would be good middle ground.

Can you elaborate why this is insufficient for you? Regarding the other libraries: I concur that there are older/unmaintained libraries that are mainly there due to inertia. 

My suggestion would be to improve the presentation on the Wiki page. I already starte this by replacing the redundant links with a short project description. I also see value in providing a star count badge and the latest release date to help users vet the projects.

AlexSkrypnyk commented:
>we don't have enough resources to curate a vetted list of third party libraries and I want to avoid users getting this impression by including them in the official documentation. 

But they are included:

<img width="752" alt="image" src="https://github.com/user-attachments/assets/ea91ba87-970e-4a74-b1e8-fc731992d1b3" />

So it looks to me that you are including some but not the others, hence my question.
But I will rest my case. I think you did not even see the quality of my library and the number of features it brings.

AlexSkrypnyk commented:
Maybe you could put a link to the wiki on the official documentation page so people reading the official documentation could see that link instead of a list of outdated libraries.

martin-schulze-vireso commented:
> Maybe you could put a link to the wiki on the official documentation page so people reading the official documentation could see that link instead of a list of outdated libraries.

Well this is exactly what this PR does now. If you are okay with that, I can mege it right away and the documentation for latest will change.

martin-schulze-vireso commented:
There you go, I am glad that we could resolve this.  Sorry for the long delay.
